### PR TITLE
 Coding aptitude assessment challenge: Task 4 - remove floating point example

### PR DESCRIPTION
**Pull request by Ngoakor12**
Related issues: [please specify]

## Description:
- Fixed the previous example `task4(1.2, 2, 3.5)` which was causing a mathematical error because its sides are those of an invalid triangle according to [the internet](https://e-gmat.com/blogs/properties-of-triangles/#:~:text=The%20sum%20of%20the%20lengths%20of%20any%20two%20sides%20of%20a%20triangle%20is%20always%20greater%20than%20the%20length%20of%20the%20third%20side) and heron's formula only works on valid triangles
- Added some usage examples according to the automarker and to be consistent with the other tasks

## I solemnly swear that:

- [x] I ran the hugo server and looked at my changed in the browser with my own eyes
- [x] I ran the linter and there were no errors
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

File: content/coding_aptitude_assessment/coding_challenges/task_4/_index.md [hunk 1]
```diff
@@ -2,28 +2,34 @@
...
+    - coding_aptitude_assessment/coding_challenges/how_to_name_files
 protect_main_branch: false
 ready: true
 submission_type: repo
-title: 'Coding aptitude assessment challenge: Task 4'
+title: "Coding aptitude assessment challenge: Task 4"
 ---
 
-Write a function that takes in three numbers. These numbers represent the lengths of the sides of a triangle. The function should return the area of a triangle.
+Write a function that takes in three numbers. These numbers represent the lengths of the sides of a triangle. The function should return the area of a triangle to the nearest whole number.
```
Comment (sheenarbw) [inline content/coding_aptitude_assessment/coding_challenges/task_4/_index.md:16]: This one doesn't match the Java implementation. Here is the code for the java perfect project:

```
public class Task4 {
    public static double task4(int sideA, int sideB, int sideC) {
        double sideLength = 0.5 * (sideA + sideB + sideC);
        double area = sideLength * (sideLength - sideA) * (sideLength - sideB) * (sideLength - sideC);

        return Math.sqrt(area);
    }
}
```

**Review by sheenarbw — CHANGES_REQUESTED** [review]
Not aligned with the Java Perfect project.

If you need help getting all the versions the same then please rach out to @ruddy-m for Java support. And of course it would be good to check the python version as well

File: content/coding_aptitude_assessment/coding_challenges/task_4/_index.md (hunk 1 above)
Comment (Ngoakor12) [reply content/coding_aptitude_assessment/coding_challenges/task_4/_index.md:16]: I'm happy to remove it. I was just fixing the example that was causing errors

**Review by sheenarbw — CHANGES_REQUESTED** [review]
This doesn't match the Java implementation. So it can't be merged without breaking stuff.

Either:
1. Update the js automarker config so it works with floating point numbers OR
2. update the java automarker config so that it follows these instructions.

This really can't be merged witout breaking something else.

**Review by sheenarbw — APPROVED** [review]

**Pull request merged**